WcaOnRails/config/routes.rb

```diff
@@ -51,6 +51,8 @@
   patch 'competitions/:competition_id/report' => 'delegate_reports#update'
 
   get 'competitions/:id/edit/admin' => 'competitions#admin_edit', as: :admin_edit_competition
+  get 'competitions/:id/edit/events' => 'competitions#edit_events', as: :edit_competition_events
+  patch 'competitions/:id/save/events' => 'competitions#save_events', as: :save_competition_events
```

Comment:
I'm not a REST fanatic, but I think the order of these last 2 pieces of the url should be switched. Something like:

```ruby
get 'competitions/:id/events/edit' => 'competitions#edit_events', as: :edit_competition_events
patch 'competitions/:id/events' => 'competitions#update_events', as: :update_competition_events
```

I believe that's more consistent with how our other edit/update routes work.